config/locales/javascript/javascript.en.yml

```diff
@@ -282,6 +282,11 @@ en:
         one: "<%= count %> Reshare"
         other: "<%= count %> Reshares"
 
+      comments:
+        zero: "<%= count %> comments"
```

Comment:
I didn't “move” from “No” to “0”. The old translations for the reaction counts in `config/locales/diaspora/en.yml` were using “No” instead of “0” but the removed translations in `config/locales/javascript/javascript.en.yml` as well as the translations for reshares and likes were using “0”, so I did the same when adding the comment count translations.